tests/end2end/features/downloads.feature

```diff
@@ -126,6 +126,14 @@ Feature: Downloading things from a website.
         Then "cancelled" should be logged
         And "cancelled" should be logged
 
+    # https://github.com/The-Compiler/qutebrowser/issues/1535
+    Scenario: Cancelling a MHTML download (issue 1535)
+        When I open data/downloads/issue1535.html
+        And I run :download --mhtml
+        And I wait 1s
```

Comment:
It failed on my machine with "no downloads running", and there was a bit of load in the background, so I attributed it to a race condition between *scanning the page and starting a download* and `:download-cancel`, but it should work without (if qutebrowser is fast enough :wink:)